brontology/graph/base/node.py

```diff
@@ -10,10 +10,12 @@ class Node(Generic[L, C]):
     """A graph node. Defined by its incoming and outgoing links."""
 
     content: C
+    id: str | None
 
-    def __init__(self, content: C = NO_CONTENT) -> None:
+    def __init__(self, content: C = NO_CONTENT, id_: str | None = None) -> None:
         if content is not NO_CONTENT:
             self.content = content
+        self.id = id_
```

Comment:
Die nodes haben jetzt eine ID damit sie einfacher von der database auseinandergehalten werden können. Aktuell ist das einfach die Reihenfolge, in der sie erstellt wurden.